test/TestEnvironment.Docker.Tests/DockerEnvironmentTests.cs

```diff
@@ -31,6 +31,7 @@ public async Task CreateDockerEnvironment()
                 .AddMssqlContainer("my-mssql", "HelloK11tt_0")
                 .AddMongoContainer("my-mongo")
 #endif
+                .AddFromDockerfile("from-file", "Dockerfile", containerWaiter: new HttpContainerWaiter("/", httpPort: 8080))
```

Comment:
Why not tested with reuseContainer: true ?

Reply:
agree